src/store/store.js

```diff
@@ -0,0 +1,5 @@
+import { createStore } from "redux";
+import { listReducer } from "./reducer";
+
+const store = createStore(listReducer);
```

Comment:
I would have preferred that you used Redux Toolkit instead of the older version of Redux but it's good to know both.